brew-script/action.yml

```diff
@@ -13,5 +13,19 @@ inputs:
     description: Ruby script to execute
     required: true
 runs:
-  using: node12
-  main: main.js
+  using: composite
+  steps:
+    - run: |
+        cat << 'EOF' > /tmp/brew-script-$GITHUB_SHA
+        ${{inputs.script}}
+        EOF
+      shell: bash
+    - run: brew ruby /tmp/brew-script-$GITHUB_SHA
+      shell: bash
+      env:
+        HOMEBREW_GITHUB_API_TOKEN: ${{inputs.token}}
+        HOMEBREW_NO_ENV_FILTERING: 1
+        HOMEBREW_NO_AUTO_UPDATE: 1
+        HOMEBREW_NO_ANALYTICS: 1
```

Comment:
Why set this?

Reply:
Could remove it if you think it isn't needed. Just thought we don't want analytics from this Action.

Reply:
Yeh, we don't want to disable analytics anywhere really.

Reply:
https://github.com/Homebrew/actions/commit/faee221d871037213f8d0e936fb113246b003539

Reply:
Thanks on both!